app/Http/Requests/Settings/SetAlbumDecorationRequest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Http\Requests\Settings;
+
+use App\Contracts\Http\Requests\RequestAttribute;
+use App\Http\Requests\BaseApiRequest;
+use App\Models\Configs;
+use App\Policies\SettingsPolicy;
+use Illuminate\Support\Facades\Gate;
+use Illuminate\Validation\Rule;
+
+class SetAlbumDecorationRequest extends BaseApiRequest
+{
+	protected string $albumDecoration;
+	protected string $albumDecorationOrientation;
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function authorize(): bool
+	{
+		return Gate::check(SettingsPolicy::CAN_EDIT, Configs::class);
+	}
+
+	public function rules(): array
+	{
+		return [RequestAttribute::ALBUM_DECORATION_ATTRIBUTE => [
+			'required',
+			'string',
+			Rule::in(['none', 'original', 'album', 'photo', 'all']),
```

Comment:
Use an Enum instead. See `ColumnSortingType.php` for example and `SetSortingSettingsRequest.php`